src/discord_utils.py

```diff
@@ -31,6 +31,26 @@ async def try_set_pin(message: Message, pin_state: bool) -> None:
         print("Altering message pin state failed:", e)
 
 
+def attachment_local_filepath(message_id: int, attachment: Attachment) -> str:
+    """Build local filepath with provided Message ID & Attachment object for saving Attachments locally"""
+    filepath = path.join(
+        config.attachment_directory_filepath,
+        "{}.{}{}".format(
+            message_id,
+            attachment.id,
+            os.path.splitext(attachment.filename)[1],
+        ),
+    )
+    return filepath
+
+
+def is_valid_media(attachment_content_type: str) -> bool:
+    return attachment_content_type is not None and (
+        (attachment_content_type.startswith("audio"))
+        or (attachment_content_type.startswith("video"))
```

Comment:
These extra parentheses aren't necessary as they only surround a single entity.

```suggestion
        attachment_content_type.startswith("audio")
        or attachment_content_type.startswith("video")
```